less/_components/_commons.less

```diff
@@ -0,0 +1,227 @@
+/*!
+ * Copyright (c) 2021 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Eric Poirier <eric.poirier@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+.featured-section{
+    padding: 60px 0 60px 0;
+    background-position: 0 100%;
+    h2{
+      font-size: 18px;
+      margin-bottom: 30px;
+      color: @brand-primary;
+      margin-top:0;
+      text-transform: uppercase;
+      font-weight: 400;
+    }
+    .feather {
+      width: 40px;
+      height: 40px;
+    }
+    ul li {
+        font-size: 14px;
+        margin-bottom:15px;
+    }
+  }
+
+.featured-section-prefix {
+    .small {
+        color:@brand-primary;
+        text-transform: uppercase;
+        margin-bottom:10px;
+        font-size:14px; 
+    }
+    .powered-by-description {
+        font-size: 24px;
+        margin-bottom:20px;
+    }
+}
+
+.offwhite-bg-shadow, .white-bg-shadow {
+  padding: 40px;
+  box-shadow: 6px 6px 10px 0px rgba(0,0,0,0.25);
+}
+
+.offwhite-bg-shadow {
+  background-color:#f9f9f9;
+}
+
+.white-bg-shadow {
+  background-color:#fff;
+}
+
+.featured-section-webinar-series-2 {
+  background-color:@white;
+  padding-top:60px;
+  h2 {
+    margin-bottom:20px;
+  }
+  .webinar-series-2-date {
+    color:@brand-primary;
+    font-size:24px;
+  }
+  .webinar-series-2-description {
+    font-size: 16px;
+    margin-bottom:25px;
+    margin-top:20px;
+  }
+  .webinar-series-2-buttons {
+    li a {
+      width:225px;
+      @media (max-width: @screen-xs-max) {
+        margin-bottom:30px;
+      }
+    }
+  }
+}
+
+.featured-section-webinar-series-2-home {
+  background-color:#f9f9f9;
```

Comment:
we should make this a var as we use it a few times and it seems to be a consistent branding thing